src/urdf.cpp

```diff
@@ -112,10 +112,15 @@ rbd::Joint::Type rbdynFromUrdfJoint(const std::string & type, bool hasSphericalS
   else if(type == "floating")
   {
     if (hasSphericalSuffix)
-      return rbd::Joint::Spherical;
+      {
+        std::cerr << "hasSphericalSuffix is deprecated and will be deleted in future release. Please use type=ball instead (style of sdf format)" << std::endl;
```

Comment:
indentation is broken here,there is also no plan to deprecate spherical suffix in the future